ddp-workspace/projects/ddp-atcp/src/app/services/dataAccess.service.ts

```diff
@@ -1,29 +1,76 @@
-import { Injectable } from '@angular/core';
-import { DataAccessServiceAgent } from './serviceAgents/dataAccessServiceAgent.service';
-import { Observable, throwError } from 'rxjs';
-import { catchError, map } from 'rxjs/operators';
-import { DataAccessRequestSuccessResult } from '../models/dataAccessRequestSuccessResult';
-import { DataAccessRequestError } from '../models/dataAccessRequestError';
+import { Inject, Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { concatMap } from 'rxjs/operators';
+
+import { ConfigurationService, LoggingService } from 'ddp-sdk';
+
 import { DataAccessParameters } from '../models/dataAccessParameters';
 
-@Injectable()
+declare global {
+  interface Window {
+    DDP_ENV: {
+      [key: string]: any;
+      darUrl?: string;
+    };
+  }
+}
```

Comment:
Instead of it, I'd recommend store `darUrl` in the ConfigurarionService, like here:
https://github.com/broadinstitute/ddp-angular/blob/develop/ddp-workspace/projects/ddp-angio/src/app/app.module.ts#L87

Reply:
Is it really necessary? Do we expect to have this kind of feature (or related one) in other studies as well?

Reply:
Okay, I don't insist in my suggestion, it's up to you.